### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by saagarjha**
These fell out of additional manual testing and user feedback for some recent changes.

### CODE REVIEW

File: app/RootsTableViewController.m [hunk 1]
```diff
@@ -90,7 +90,11 @@ - (void)documentPicker:(UIDocumentPickerViewController *)controller didPickDocum
 
     dispatch_async(dispatch_get_global_queue(QOS_CLASS_USER_INITIATED, 0), ^{
         NSError *error;
-        BOOL success = [Roots.instance importRootFromArchive:urls[0] name:name error:&error progressReporter:progressVC];
+        BOOL success = NO;
+        if ([url startAccessingSecurityScopedResource]) {
```
Comment (tbodt) [inline app/RootsTableViewController.m:94]: error is used uninitialized if this is false

Comment (saagarjha) [reply app/RootsTableViewController.m:94]: It would be nil which should be fine in this case?

Comment (tbodt) [reply app/RootsTableViewController.m:94]: it wouldn't, but realized that success is false so this is fine. Still, should we show something to the user if this fails? Not sure what failure here actually means though

Comment (saagarjha) [reply app/RootsTableViewController.m:94]: Oh I just spotted the `error != nil` check, we can delete that

Comment (tbodt) [reply app/RootsTableViewController.m:94]: Not sure if importRootFromArchive can maybe return false with error null?

Comment (saagarjha) [reply app/RootsTableViewController.m:94]: In any case when something fails we should tell the user that the import didn't succeed, even if we don't have any additional error information

Comment (tbodt) [reply app/RootsTableViewController.m:94]: Makes sense, but we should do that without possibly reading uninitialized data

Comment (saagarjha) [reply app/RootsTableViewController.m:94]: I'm not seeing the uninitialized read

Comment (tbodt) [reply app/RootsTableViewController.m:94]: startAccessingSecurityScopedResource returns no, then success is no, but error was never set and will be read in the if below, and then maybe by presentError

Comment (saagarjha) [reply app/RootsTableViewController.m:94]: Right, it will be read as nil which is fine here

Comment (tbodt) [reply app/RootsTableViewController.m:94]: Today I learned objective-c auto initializes object pointer variables to nil https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/WorkingwithObjects/WorkingwithObjects.html#//apple_ref/doc/uid/TP40011210-CH4-SW22

**Pull request merged**